packages/core/package.json

```diff
@@ -40,32 +40,32 @@
   },
   "dependencies": {
     "@loadable/component": "5.16.4",
-    "@mdx-js/loader": "2.3.0",
-    "@mdx-js/mdx": "2.3.0",
-    "@mdx-js/react": "2.3.0",
+    "@mdx-js/loader": "3.0.1",
+    "@mdx-js/mdx": "3.0.1",
```

Comment:
I'm worried that mdx v3 and remark v15 contains some breaking changes and may break the user code / plugins.

Reply:
I believe the [mdx-rs](https://github.com/web-infra-dev/mdx-rs) of Rspress use the mdx v2 syntax, @sanyuan0704 can you confirm this?

Reply:
Looks like [mdx v3 updates the rehype and remark](https://mdxjs.com/migrating/v3/#update-plugins), which contains breaking changes mainly in typings. This is indeed a breaking change for Rspress users, but migration is easy. Just update the rehype and remark plugins.